miso-web/src/main/webapp/scripts/project_ajax.js

```diff
@@ -193,7 +192,7 @@ Project.ui = {
             "bJQueryUI": true,
             "iDisplayLength": 25,
             "aaSorting": [
-              [0, "desc"]
+              [0, "asc"]
```

Comment:
Why the change to `asc`? I would think that the lab would want to see more recent projects first.